angular-swfobject.js

```diff
@@ -60,23 +60,21 @@ angular.module('swfobject', [])
                   $interval.cancel(loadCheckInterval);
                   loadCheckInterval = null;
                   //Execute function
-                  fn();
+                  fn({evt: evt});
                 }
               }, 1500);
             }
           }, 200);
         }
 
-        //This function is invoked by SWFObject once the <object> has been created
+        //a callback function that is called on both success or failure of creating a Flash plug-in <object> on the page (SWFObject 2.2+)
```

Comment:
You could probably remove this comment altogether and just link to the docs: https://code.google.com/p/swfobject/wiki/api#swfobject.registerObject(objectIdStr,_swfVersionStr,_xiSwfUrlStr